src/schema/users.ts

```diff
@@ -27,11 +31,29 @@ import { getSearchQuery } from './common';
 import { ActiveView } from '../entity/ActiveView';
 import graphorm from '../graphorm';
 import { GraphQLResolveInfo } from 'graphql';
+import { isNullOrUndefined } from '../common/object';
+
+export interface GQLUpdateUserInput {
+  name: string;
+  username?: string;
+  bio?: string;
+  company?: string;
+  title: string;
+  twitter?: string;
+  github?: string;
+  hashnode?: string;
+  portfolio?: string;
+}
+
+interface GQLUserParameters {
+  data: GQLUpdateUserInput;
+  upload: FileUpload;
+}
 
 export interface GQLUser {
   id: string;
   name: string;
-  image: string;
+  image?: string;
```

Comment:
Why is this optional all of a sudden?

Reply:
In our actual User entity, this is optional. We don't handle the fallback in our response.